core/mm/core_mmu.c

```diff
@@ -2691,10 +2677,18 @@ static void __maybe_unused carve_out_core_mem(paddr_t pa, paddr_t end_pa)
 
 void core_mmu_init_phys_mem(void)
 {
-	paddr_t ps = 0;
-	size_t size = 0;
+	if (IS_ENABLED(CFG_NS_VIRTUALIZATION)) {
+		paddr_t b1 = 0;
+		paddr_size_t s1 = 0;
 
-	if (!IS_ENABLED(CFG_NS_VIRTUALIZATION)) {
+		static_assert(ARRAY_SIZE(secure_only) <= 2);
+		if (ARRAY_SIZE(secure_only) == 2) {
```

Comment:
An an empty line above.

+ I think it would be worth restoring the info level trace message
`IMSG("Initializing virtualization support");`

Reply:
Sure, I'll add an empty line.

We don't want it in this file. We're adding a new debug level message in "core: arm: add CFG_NS_VIRTUALIZATION boot log". Do you mean this print is important enough that the level should be info rather than debug? Debug level is the right level IMO.

Reply:
No, i'm fine if the old trace message is  removed.